sct.py

```diff
@@ -1184,8 +1184,18 @@ def run_test(argv, backend, config, logdir):
     sys.stdout = OutputLogger(logfile, sys.stdout)
     sys.stderr = OutputLogger(logfile, sys.stderr)
 
-    unittest.main(module=None, argv=['python -m unittest', argv],
-                  failfast=False, buffer=False, catchbreak=True, testLoader=SctLoader())
+    if '::' in argv:
```

Comment:
Do we really need backwards compatible unit test command? I would pass all arguments here directly to pytest and made the interface more pytest related

Reply:
it's gonna be changes in hundreds of file

Reply:
all of sct pipeline runs end up in this call

all of the function are like: `longevity_test.LongevityTest.test_custom_something`
and that is mentioned in hundreds of pipeline files, which I don't yet want to change to pytest format.

it's gonna be a nightmare to backport case into release with such changes